test/ttmlir/Dialect/TTIR/slice/slice_tests_negative.mlir

```diff
@@ -0,0 +1,191 @@
+// RUN: not ttmlir-opt --split-input-file %s 2>&1 | FileCheck %s
+// Negative tests for slice operation
+
+#any_device_tile = #tt.operand_constraint<dram|l1|tile|any_device_tile>
+module attributes {} {
+  // Verify that the parsing fails if the begins attribute is not a 3D tensor
```

Comment:
nit: To be the consistent with the rest of the file, please move this comment above the module

Reply:
Will do, I don't know how it ended there :D